packages/next-frontend/src/components/step/index.tsx

```diff
@@ -31,44 +36,37 @@ export function Step({
         matchChildByType(child, StepContent),
     ) as ReactElement;
 
+    useEffect(() => {
+        animate({
+            height: (open ? wrapperRef?.current?.offsetHeight : 83) + "px",
```

Comment:
No way to use auto here? Not particularly loving the explicit pixels...

Reply:
There's an issue while trying to animate between number and `auto` here:
>You are trying to animate between an AnimatedValue and an AnimatedString. In React Spring.